### ISSUE TITLE
Avoid hard-coding minor revisions

### ISSUE THREAD

**Issue by jadonk** [RobertCNelson/Bootloader-Builder#6]
Is it possible to adjust evaluation of minor revisions such that older software can still run on newer boards? Case in point: BWA3.  Can it not be BWAx or even BWxx?

**Comment by RobertCNelson** [comment]
@jadonk yeah i can do that, but let's add it to the list, that way we can say "BWxx" or "BWAx" is taken..

**Comment by RobertCNelson** [comment]
@jadonk as of:

https://github.com/RobertCNelson/Bootloader-Builder/commit/01786b3c3408e65b260375054debfe7897dff796

```
static inline int board_is_bbbw(void)
{
	return board_is_bone_lt() && !strncmp(board_ti_get_rev(), "BW", 2);
}
```
